static/index.html

```diff
@@ -52,17 +52,15 @@
       </div>
     </div>
     <div class="heading">
-      <img src="/static/images/globe.svg" class="logo" alt="World driven logo" height="60" width="60">
-      <h1>World Driven</h1>
+      <img src="/static/images/logo_world_driven.svg" class="logo" alt="World driven logo">
+      
     </div>
     <center><h6>Trustable service landscape</h6></center>
-    <h2>Phase One: Dynamic Code Ownership</h2>
-
-    <h3>The Challenge</h3>
+    <h2 class='phases'>Phase One: Dynamic Code Ownership</h2>
 
+    <h3 class="subheading">The Challenge</h3>
```

Comment:
This and the above 'phases' class is a nice idea, giving them different colours that are part of the colour scheme, but you've only done it on the first phase and the first subheading - This should be made consistent throughout the page.